ACPs/77-reinventing-subnets/README.md

```diff
@@ -179,24 +184,26 @@ The `SubnetValidatorWeightMessage` is specified as an `AddressedCall` with the f
 
 ### New P-Chain Transaction Types
 
-Both before and after this ACP, to create a Subnet, a `CreateSubnetTx` must be issued on the P-Chain. This transaction includes an `Owner` field which defines the key that today can be used to authorize any validator set additions (`AddSubnetValidatorTx`) or removals (`RemoveSubnetValidatorTx`). 
+Both before and after this ACP, to create a Subnet or an L1, a `CreateSubnetTx` must be issued on the P-Chain. This transaction includes an `Owner` field which defines the key that today can be used to authorize any validator set additions (`AddSubnetValidatorTx`) or removals (`RemoveSubnetValidatorTx`).
```

Comment:
I believe it's considered a Subnet post-CreateSubnetTx and an L1 post-ConvertSubnetToL1Tx

```suggestion
To create a Subnet, a `CreateSubnetTx` must be issued on the P-Chain. This transaction includes an `Owner` field which defines the key that today can be used to authorize any validator set additions (`AddSubnetValidatorTx`) or removals (`RemoveSubnetValidatorTx`).
```

Reply:
this should be kept bc this part of the flow remains the same pre and post acp-77